### ISSUE TITLE
embed_doc does not work

### ISSUE THREAD

**Issue by shamidreza** [facebookresearch/StarSpace#283]
I want to use embed_doc on a trained mode
`embed_doc model       `         
and instead on a stdin, it immediately closes:                      
```
Start to load a trained starspace model.
STARSPACE-2018-2
Model loaded.
Must provide base labels when label is featured.
```
What is the issue here?

**Comment by Magdiel3** [comment]
Hi, which trainMode did you used for that model?

**Comment by shamidreza** [comment]
I use trainMode 0 and labelDoc.
I managed to comment out this line: https://github.com/facebookresearch/StarSpace/blob/8aee0a950aa607c023e5c91cff518bec335b5df5/src/starspace.cpp#L130

and recompile. This solved the issue that it needed me to provide base docs, but does not accept it via sending arguments.

For embed_doc, we don't really need base labels as I understand. Correct?

**Comment by nandanrao** [comment]
Indeed, line 130 in starspace.cpp was added in commit 8aee0a950aa607c023e5c91cff518bec335b5df5. But in theory that commit has nothing to do with training. Not sure what the idea with that was?
